app/jobs/send_periodic_invites_job.rb

```diff
@@ -18,7 +18,7 @@ def perform
 
   def send_invite(rdv_context)
     last_sent_invitation = rdv_context.last_sent_invitation
```

Comment:
nitpick: On pourrait peut-être aussi rajouter le scope `sent` au moment de récupérer les invitations à la ligne 11 pour itérer sur moins de `rdv_contexts` en ne traitant pas ceux qui n'ont pas d'invitations envoyées